halo2-ecc/src/bn254/pairing.rs

```diff
@@ -222,8 +222,9 @@ pub fn miller_loop_BN<F: PrimeField, C>(
     Q: &EcPoint<F, FqPoint<F>>,
     P: &EcPoint<F, FpPoint<F>>,
     pseudo_binary_encoding: &[i8],
-) -> FqPoint<F> 
- where C: CurveAffine<Base = Fp2Chip<F>>,
+) -> FqPoint<F>
+where
```

Comment:
in the comment "Q = (x, y) is a point in E(Fp2)
// - P is a point in E(Fp)" 
so can we only use one C to represent two type points' curves?

Reply:
yesterday i considered two types C1, C2 for Fp2 and Fp .

Reply:
oh, only for Q affine will be ok as only double Q point inside